### ISSUE TITLE
[AVM FRITZ!Box] Received unknown command 'REFRESH' for channel outlet

### ISSUE THREAD

**Issue by NorbertHD** [openhab/openhab2-addons#4014]
When I start the openhab2 service I see these warnings in the log:
```
2018-09-24 10:25:13.261 [WARN ] [itz.handler.AVMFritzBaseThingHandler] - Received unknown command 'REFRESH' for channel outlet
2018-09-24 10:25:13.291 [WARN ] [itz.handler.AVMFritzBaseThingHandler] - Received unknown command 'REFRESH' for channel outlet
```
Is this relevant?

openHAB 2.3.0 Release, FRITZ!Box 7590 with FRITZ!OS 07.01, 2 FRITZ!DECT 200 v04.09

**Comment by kaikreuzer** [comment]
Not for you as a user, but these are two bugs in the binding:
1. REFRESH commands should be handled
2. No warnings must be logged upon unknown commands

**Comment by cweitkamp** [comment]
I submitted a to fix it (#4051.)
